### PR TITLE
[frameworks][now-static-build] Add support for Scully

### PR DESCRIPTION
**Pull request by AndyBitz**
Add support for Scully.

### CODE REVIEW

File: packages/now-static-build/src/frameworks.ts [hunk 1]
```diff
@@ -121,6 +121,14 @@ export const frameworks: Framework[] = [
       },
     ],
   },
+  {
+    name: 'Scully',
+    slug: 'scully',
+    dependency: '@scullyio/init',
+    minNodeRange: '10.x',
+    buildCommand: 'ng build && scully',
+    getOutputDirName: async () => 'dist/static',
```
Comment (styfle) [inline packages/now-static-build/src/frameworks.ts:130]: Are you sure that the output is correct?

It seems like you should read `angular.json` file as described in [this tweet]( https://mobile.twitter.com/styfle/status/1208073521190051844).

**Review by styfle — COMMENTED** [review]
It seems this uses the same algorithm as angular

File: packages/now-static-build/src/frameworks.ts (hunk 1 above)
Comment (AndyBitz) [reply packages/now-static-build/src/frameworks.ts:130]: Right, this is just the default for Scully. I think the problem is that there can be multiple projects in one angular.json file with multiple output paths. I'll need to check further

**Review by leo — APPROVED** [review]
Ship it!

**Review by styfle — APPROVED** [review]

File: packages/now-static-build/src/frameworks.ts (hunk 1 above)
Comment (leo) [reply packages/now-static-build/src/frameworks.ts:130]: https://zeit.atlassian.net/browse/PRODUCT-935

**Pull request merged**